alibi_detect/saving/loading.py

```diff
@@ -115,19 +117,16 @@ def _load_detector_config(filepath: Union[str, os.PathLike]) -> ConfigurableDete
     Parameters
     ----------
     filepath
-        Directory containing the `config.toml` file.
+        Filepath to the `config.toml` file.
 
     Returns
     -------
     The instantiated detector.
     """
-    # Load toml if needed
-    if isinstance(filepath, (str, os.PathLike)):
-        config_file = Path(filepath)
-        config_dir = config_file.parent
-        cfg = read_config(config_file)
-    else:
-        raise ValueError("`filepath` should point to a directory containing a 'config.toml' file.")
```

Comment:
why are you removing this error? Do we check somewhere else?

Reply:
I removed this since we already make these checks in `load_detector`. However, thinking again, since `_load_detector_config` will eventually become `load_detector` (when we remove legacy save/load), it makes more sense to leave this check in. I'll revert this now.